source/commands/create_command.py

```diff
@@ -0,0 +1,50 @@
+import os, sys,json
+parent_dir = os.path.abspath(os.path.join(os.path.dirname(__file__), os.pardir))
+sys.path.append(parent_dir)
+from commands.abstract_command import AbstractCommand
+from commands.keys import *
+from output.exceptions import *
+
+
+class CreateCommand(AbstractCommand):
+
+    def __init__(self , schema_path):
+        self.__validate(schema_path)  
+           
+    
+    def __validate(self,schema_path):
```

Comment:
Why you didn’t make it a static function ?

Reply:
I realized that I need it to be instance method not static because I need the ```self.``` to make the ```__load_data``` works

Reply:
So I made it private method but instance not static and I think that is not a problem

Reply:
```
    def __load_data(new_schema_path):
        try:  
            with open(new_schema_path, "r") as schema:
               return json.load(schema)
        except:
            raise ErrorLoadingJsonFile("The json file could not be loaded")

```